neo-cli/neo-cli.csproj

```diff
@@ -32,6 +32,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\neo\neo.csproj" />
+    <ProjectReference Include="..\..\neo\neo\neo.csproj" />
```

Comment:
Note that neo, are in the neo folder https://github.com/neo-project/neo/tree/master/neo